spec/appliance_console/external_httpd_authentication_spec.rb

```diff
@@ -162,4 +163,26 @@
       subject.post_activation
     end
   end
+
+  context "#configure_ipa_http_service" do
+    before do
```

Comment:
I think it looks more clean to separate what we are testing from what is not important